### PR TITLE
Dbp 944 switch chart release workflow

### PR DESCRIPTION
**Pull request by JannikBadenhop**
# Description

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: .github/workflows/helm-chart-release-on-push.yaml [hunk 1]
```diff
@@ -1,10 +1,13 @@
-name: Release DBP Moodle Chart for dev
```
Comment (maxi418) [inline .github/workflows/helm-chart-release-on-push.yaml:1]: i think the workflow should have a name

What the workflow does is to release a helm chart on push for development

File: .github/workflows/helm-chart-release-on-push.yaml [hunk 2]
```diff
@@ -23,9 +26,8 @@ jobs:
       fail_on_vulnerabilites: false
 
   release_helm:
-    uses: ./.github/workflows/chart-release.yaml
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/chart-release.yaml@7
     secrets: inherit
     with:
       chart_name: dbp-moodle
-      helm_chart_version_generation: ticket_from_branch_timestamp
```
Comment (maxi418) [inline .github/workflows/helm-chart-release-on-push.yaml:30]: the chart version should be based on the ticket number to match it with the corresponding ticket easily

File: .github/workflows/helm-chart-release-on-push.yaml [hunk 3]
```diff
@@ -23,9 +26,8 @@ jobs:
...
-      image_tag_generation: ticket_from_branch
```
Comment (maxi418) [inline .github/workflows/helm-chart-release-on-push.yaml:31]: same for image tag

Comment (JannikBadenhop) [reply .github/workflows/helm-chart-release-on-push.yaml:31]: ticket_from_branch is default here but i can make it explicit.

Comment (maxi418) [reply .github/workflows/helm-chart-release-on-push.yaml:31]: would be best i think

**Review by maxi418 — APPROVED** [review]

**Pull request merged**